### ISSUE TITLE
Compilation failure on `#[multiversion(targets = "simd")]`

### ISSUE THREAD

**Issue by Serock3** [calebzulawski/multiversion#37]
The project I am working on depends on the arrow2 crate, which uses Multiversion. I am running into compilation errors when cross-compiling for Arm Cortex-A8 architecture. The compiler output look like this

```
error[E0658]: the target feature `d32` is currently unstable
   --> /opt/cem-sdk/sysroots/cortexa8t2hf-neon-poky-linux-gnueabi/home/cargo/registry/src/github.com-1ecc6299db9ec823/arrow2-0.17.3/src/compute/aggregate/min_max.rs:116:1
    |
116 | #[multiversion(targets = "simd")]
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #44839 <https://github.com/rust-lang/rust/issues/44839> for more information
    = note: this error originates in the attribute macro `multiversion` (in Nightly builds, run with -Z macro-backtrace for more info)
```

(We do not intend to use any SIMD features)

Please let me know if there is any more information I can give you to help debug this issue, I don't know much about CPU architectures or this crate.

**Comment by calebzulawski** [comment]
Okay, this is fixable. It doesn't come up on x86 or aarch64 since those features are all stable. I should be able to fix this with a new release, no changes to arrow2.

**Comment by Serock3** [comment]
Great!

**Comment by calebzulawski** [comment]
Fixed via 91b09dc85f72ed1d6fe6d5af73bd8c08b8964658, please try upgrading to 0.7.3
